src/docs/Foundations/Layout/Grid/code.mdx

```diff
@@ -24,43 +19,24 @@ order: 1
 @import 'layouts/_l.flexy';
 ```
 
-`ml-flexy` is very close to the bootstrap flex grid system.
+### How it works
 
-To create a grid, you need two elements:
+The below example creates four equal-width columns at small, medium, large, and extra large breakpoints using our predefined grid classes. The columns are centered in the page within the parent container.
 
-- A grid: `ml-flexy`
-- Columns: `ml-flexy__col`
+<Preview path="basic" />
 
-```html
-<div class="ml-flexy">
-  <div class="ml-flexy__col">
-    col 1
-  </div>
+#### Container
+`ml-container` defines the overall grid context at the top level and applies the proper margin and our default max-width.  You can also remove this max width, by using the grid modifier class ml--container--fluid.
```

Comment:
The `ml-container` plays no role in the construction of the grid. 
Therefore, I think we should put this part in a "Highlight".

On the other hand, you could replace this part with an explanation of the `ml-flexy` class.